test/clumpr/recon_vis/test_Tree.py

```diff
@@ -0,0 +1,62 @@
+# test_Tree.py
+# Tests the Tree class and tree_format_converter.py
+import unittest
+
+from empress.clumpr.recon_vis import tree
+from empress.clumpr.recon_vis.utils import dict_to_tree
```

Comment:
Because `recon_vis.tree` is not dependent on `recon_vis.utils`, `test_Tree` should avoid importing `recon_vis.utils` as this can make the test confusing to read.